Makefile

```
@@ -0,0 +1,11 @@
+test:
+	cargo build
```

This should be
```suggestion
	cargo build --all-targets
```
Otherwise the `test_server` is not built and some tests will fail.